plugins/modules/foreman_bookmark.py

```diff
@@ -36,25 +36,6 @@
   - "ansible >= 2.3"
   - "apypie"
```

Comment:
It should be possible, to move the requrements to the doc fragment, i think.
I guess, they will even get merged. If specified here too.

Reply:
it is, but I didn't do that for the sake of not having two fragments (nailgun, apypie) for now.

Reply:
So we postpone that. OK for me.

Reply:
I guess inheriting doc fragements is not really worth it on the string level. ;)

Reply:
Yeah, I thought about that, but it didn't feel like a worthy endeavor for a one-line thing.